### PR TITLE
fibanocci python + readme

### PR DESCRIPTION
**Pull request by itsnikhil**

### CODE REVIEW

**Comment by t2013anurag** [comment]
@itsnikhil - There's a fib directory. Can you rename it to fibonacci and move your code in that directory. Also since we already have one readme for that you can remove this readme.

File: fibonacci/fib.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import future
+
+def fib(n):
+    a, b = 0, 1
+    while a < n:
+        print(a)
+        a, b = b, a+b
+fib(1000)
```
Comment (t2013anurag) [inline fibonacci/fib.py]: new line. and use https://github.com/hacktoberfest17/programming/blob/master/fibonacci/fibonacci.cpp for input/output reference.

File: fibonacci/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The following python program will output fub series
```
Comment (t2013anurag) [inline fibonacci/README.md]: remove this file.

**Comment by itsnikhil** [comment]
done

**Comment by t2013anurag** [comment]
@itsnikhil - can you move your file inside the fibonacci.

**Comment by t2013anurag** [comment]
@itsnikhil - code already exists. Closing.

**Pull request closed without merging**